packages/bottom-tabs/src/views/BottomTabBar.tsx

```diff
@@ -249,18 +260,87 @@ export default function BottomTabBar({
     descriptors,
     dimensions,
     layout,
+    tabBarPosition,
   });
 
   const tabBarBackgroundElement = tabBarBackground?.();
 
+  const tabBarPositionMap: TabBarPositionMapConfig = {
+    right: {
+      tabBarItem: {
+        height:
+          Platform.OS === ('ios' || 'android')
```

Comment:
This equality check is incorrect. `'ios' || 'android'` will always evaluate to `'ios'`, so the check is `Platform.OS === 'ios'`. Why is this condition based on iOS or not?

Reply:
This has been removed from the previous comment od using dynamic height